docs/sdk-and-tools/indexer.md

```diff
@@ -0,0 +1,139 @@
+---
+id: indexer
+title: Elasticindexer service
+---
+
+[comment]: # (mx-abstract)
+
+## Overview
+
+A MultiversX observer node can send messages over `WebSocket` to an elastic indexer service, which will process and index the data in an Elasticsearch database.
```

Comment:
add a statement here that this feature will work starting from rc/v1.6.0 version of the node?

Reply:
added